plugins/wordpress-rule-exclusions-before.conf

```
@@ -210,6 +210,16 @@ SecRule REQUEST_FILENAME "@endsWith /index.php" \
             "t:none,\
             ctl:ruleRemoveTargetById=942100;ARGS"
 
+# Cannot update page|post in WordPress due to `x-http-method-override` header
+SecRule REQUEST_FILENAME "@rx /wp-json/wp/v[0-9]+/(?:posts|pages|users)" \
+    "id:9507146,\
+    phase:1,\
+    nolog,\
+    pass,\
+    t:none,\
+    ver:'wordpress-rule-exclusions-plugin/1.0.1',\
+    setvar:'tx.restricted_headers=/content-encoding/ /proxy/ /lock-token/ /content-range/ /if/ /x-http-method/ /x-method-override/'"
```

The `restricted_headers` variable was split in two for v4.0. 
```suggestion
    setvar:'tx.restricted_headers_basic=/content-encoding/ /proxy/ /lock-token/ /content-range/ /if/ /x-http-method/ /x-method-override/'"
```